src/components/navigation/NavigationMenu.vue

```diff
@@ -122,6 +129,10 @@ import { PackageLoader } from '../../model/installing/PackageLoader';
             return this.activeGame.packageLoader === PackageLoader.BEPINEX;
         }
 
+        get canShowServerList(): boolean {
```

Comment:
This should probably read from the provider instead of be implemented here

Reply:
If the provider is simply a vue component, perhaps we should modularize the structure a bit more in a way where it can also include this logic

Reply:
I don't quite understand how the provider works, how can this check be moved to the provider?🤔
Also, hard coding a game name seems like a bad practice, but it might be solved in the future?